src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeType/ChoiceAttributeType.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AttributeBundle\Form\Type\AttributeType;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author Saša Stamenković <umpirsky@gmail.com>
+ */
+class ChoiceAttributeType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getParent()
+    {
+        return 'choice';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults(array(
+            'label' => false,
+        ));
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return 'sylius_attribute_type_choice';
```

Comment:
Ouch, this can be a bit confusing. Maybe we could figure out a more appropriate name for this attribute type. I think Akeneo is calling that "single select" and "multi select", wdyt?